cv32e40s/tb/uvmt/uvmt_cv32e40s_triggers/vplan_coverage.txt

```diff
@@ -0,0 +1,141 @@
+
+The debug vplan is written for simulation verification, and it is therefor not always clear what to verify when using assertions.
```

Comment:
This is excellent additional information, but it would be better as comments near the relevant assertions in the asserts file.

Reply:
Yess!